Evergreen/Apps/Get-JetBrainsIntelliJIDEA.ps1

```diff
@@ -0,0 +1,54 @@
+Function Get-JetBrainsIntelliJIDEA {
+    <#
+        .SYNOPSIS
+            Get the current version and download URLs for each edition of IntelliJ IDEA.
+
+        .NOTES
+            Author: Andrew Cooper
+            Twitter: @adotcoop
+    #>
+    [OutputType([System.Management.Automation.PSObject])]
+    [CmdletBinding(SupportsShouldProcess = $False)]
+    param (
+        [Parameter(Mandatory = $False, Position = 0)]
+        [ValidateNotNull()]
+        [System.Management.Automation.PSObject]
+        $res = (Get-FunctionResource -AppName ("$($MyInvocation.MyCommand)".Split("-"))[1]),
+
+        [Parameter(Mandatory = $False, Position = 1)]
+        [ValidateNotNull()]
+        [System.String] $Filter
+    )
+
+    ForEach ($Edition in $res.Get.Update.Editions.GetEnumerator()) {
+
+        # Build the update uri based on the edition
+        $uri = $res.Get.Update.Uri -replace $res.Get.Update.ReplaceEdition, $Edition.Value
+
+        # Query the Jetbrains URI to get the JSON
+        try {
+            $updateFeed = Invoke-RestMethodWrapper -Uri $uri
```

Comment:
`Invoke-RestMethodWrapper` should already handle a failure to read the URI so that you don't need to add your own `try/catch` block here.